tests/harvest/test_scenarios.py

```
@@ -216,12 +223,17 @@
-        )
-        length_store = len_store(travharv.target_store)
-        travharv.process()
+
+        try:
+            travharv = HarvestService(
+                config,
+                store,
+            )
+            length_store = len_store(travharv.target_store)
+            travharv.process()
+        except Exception as e:
```

Ruff

Do not catch blind exception: `Exception`

(BLE001)